src/deps_log.cc

```diff
@@ -20,6 +20,9 @@
 #include <string.h>
 #ifndef _WIN32
 #include <unistd.h>
+#elif (_MSC_VER < 1900)
```

Comment:
You'd need the `defined(_MSC_VER)` here, too, as MinGW defines `_WIN32`, but not `_MSC_VER`.

Reply:
My understanding is that MinGW needs these defines (but it does not need the strtoll) . 
Could somebody confirm?

Reply:
MinGW doesn't need them, otherwise Ninja wouldn't have compiled with it all the time ;)

I've added a MinGW build on AppVeyor, can you rebase your PR on the master branch?

Reply:
Make sense. Done.